docs-src/python/chapter-durable-execution/how-to-replay-a-workflow-execution-in-go.md

```diff
@@ -0,0 +1,69 @@
+---
+id: how-to-replay-a-workflow-execution-in-go
```

Comment:
This entire file from filename to contents has Go content

Ok, the next one too. I'll stop reviewing here. Can you mark this as draft if not ready for review yet?